### ISSUE TITLE
Broken Kafka Streaming ingestion page

### ISSUE THREAD

**Issue by t0mpere** [pinot-contrib/pinot-docs#338]
https://docs.pinot.apache.org/basics/data-import/pinot-stream-ingestion/import-from-apache-kafka 

<img width="854" alt="image" src="https://github.com/pinot-contrib/pinot-docs/assets/24777707/2eb2f854-4e50-49e6-b48e-e603c4191fd0">

**Comment by Jackie-Jiang** [comment]
Thanks for reporting this! Do you want to help contribute a fix?

**Comment by t0mpere** [comment]
Yes, of course. Is there a way for me to edit and preview? Or, even better, have access to the Gitbook UI? It would make my life easier for these kinds of things.

**Comment by Jackie-Jiang** [comment]
I guess it will be similar to #331
